protocol/build.sbt

```diff
@@ -19,8 +19,8 @@
 // JSON DEPENDENCIES
 //
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play-json" % "2.3.10" excludeAll( // Apache v2
-      ExclusionRule(organization = "com.fasterxml.jackson.core")
+  "com.typesafe.play" %% "play-json" % "2.5.9" excludeAll( // Apache v2
+      //ExclusionRule(organization = "com.fasterxml.jackson.core")
```

Comment:
I have been using this with Spark 2.0.2 without any problem either on submit or eval. Any specific way to reproduce the issues you are mentioning ?